src/axiosConfig.js

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
 
 const { REACT_APP_BASE_URL } = process.env;
+window.localStorage.setItem('token', 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NiwidXNlcm5hbWUiOiJKb3NoX01vcmFjaGEiLCJleHAiOjE1ODcwMjk1NDB9.RciaCNVDgbQVr6ZZv0PkamPh0MOEUx8LN1y7PbEsCq0');
```

Comment:
I don' t think this is necessary anymore since login functionality is already merged ... Please delete this before your PR is merged